subschemas/substrate-chain/src/typeDefs/parachains.ts

```diff
@@ -24,7 +24,7 @@ export default /* GraphQL */ `
   }
   type Lease {
     period: String
-    blockTime: String
+    blockTime: [String]!
```

Comment:
Nitpick:
We can use `[String!]!` to indicate that individual items of the list can't also be `null`.
This is just to have a bit more strict types.

Reply:
Yeah, you're right, not nitpick at all - it's more correct 👍